widgets/mpd.lua

```diff
@@ -37,6 +37,7 @@ local function worker(args)
     local music_dir   = args.music_dir or os.getenv("HOME") .. "/Music"
     local cover_size  = args.cover_size or 100
     local default_art = args.default_art or ""
+    local show_notifications = args.show_notifications == true
```

Comment:
Not the same pattern like the next line because `args.show_notifications or true` would still be evaluated to `true` if I set `show_notifications = false` in the input table for the widget.